### ISSUE TITLE
Add support for MySQL

### ISSUE THREAD

**Issue by zxjon22** [zxjon22/trac2gitea#9]
... since our production instance of `Gitea` uses it.

**Comment by haumacher** [comment]
[migmedia/trac2gitea](https://github.com/migmedia/trac2gitea) has worked on MySQL support. Have you already looked a these changes?

**Comment by haumacher** [comment]
Unfortunately, the fork mentioned above only changed the SQL syntax in-line. So the new version only works with MySQL but no longer with sqlite - if one only wants to convert the own Trac good enough, not for a migration tool for all...

Isn't there an abstraction layer at least within gitea that allows to manipulate the data structures without reformulating all the SQL in all dialects again?

**Comment by zxjon22** [comment]
Yeah I looked at [migmedia/trac2gitea](https://github.com/migmedia/trac2gitea) but noticed it's not been updated for a couple of years.

There are a few things that don't work properly anymore in the original version due to changes in Gitea. I've been trying the fix them up this week.

I've only got a few repos to migrate so am trying to do as little work as possible - I know pretty much zero re: Gitea internals. I'll probably go back to that fork to get a branch here working with MySQL. Don't fancy abstracting stuff out to support multiple databases ;)

I did have a look at the Gitea APIs, but you can't set things like issue creation date which is a deal breaker for me.
